partner.py

```diff
@@ -12,10 +12,14 @@
 
 
 class Partner:
+
     def __init__(self, partner_id):
 
         self.id = partner_id
 
+        self.batch_size_single = constants.DEFAULT_BATCH_SIZE
+        self.batch_size_multi = constants.DEFAULT_BATCH_SIZE
```

Comment:
batch_size_single_partner / batch_size_multi_parnter

Reply:
Do we have to assign a value to them here since `compute_batch_sizes`() will overwrite these values ?